pycbc/events/coinc_rate.py

```diff
@@ -0,0 +1,156 @@
+#
+# =============================================================================
+#
+#                                   Preamble
+#
+# =============================================================================
+#
+""" This modules contains functions for calculating and manipulating
```

Comment:
modules -> module 
i think 'calculating expected rates of noise and signal coincidences' would be more accurate